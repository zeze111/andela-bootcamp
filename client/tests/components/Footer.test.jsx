import React from 'react';

import Footer from '../../components/Footer';

describe('Test for footer', () => {
  it('', () => {
    const wrapper = shallow(<Footer /> );
    expect(wrapper).toMatchSnapshot()
  });
});
