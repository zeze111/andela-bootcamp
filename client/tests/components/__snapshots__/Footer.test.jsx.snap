// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for footer  1`] = `
<footer
  className="page-footer nav-color"
>
  <div
    className="container"
  >
    <div
      className="row remove-margin-bottom"
    >
      <div
        className="footer-center"
      >
        <i
          className="material-icons col s3 l1 right-align"
        >
           local_dining 
        </i>
        <p
          className="col s6 l3 left-align footer-style2 footer-style no-top"
        >
           MoreRecipes
        </p>
      </div>
    </div>
  </div>
  <div
    className="footer-copyright footer-style2"
  >
    <div
      className="container"
    >
      <div
        className="row remove-margin-bottom"
      >
        <p
          className="col s3 m1 l1 push-s1 footer-style2"
        >
           Made by 
        </p>
        <p
          className="col s6 m3 l3 teal-text text-lighten-3 left-align"
        >
          Osaze Edo-Osagie
        </p>
      </div>
    </div>
  </div>
</footer>
`;
